components/features/wallet/hooks/use-asset-actions.ts

```
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import { useAsyncFn } from "react-use";
+
+export const useAssetActions = () => {
+	const [error, setError] = useState<string | null>(null);
+
+	const [{ loading: loadingSellStart, value: sellStartData }, sellStart] =
+		useAsyncFn(async (body: SellAssetStartBody) => {
+			setError(null);
+			try {
+				const res = await sellAssetStart(body);
+				debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)